docs/public/_redirects

```diff
@@ -17,6 +17,7 @@
 /x/license/ https://mui.com/store/legal/mui-x-eula/ 301
 /x/react-data-grid/group-pivot/ /x/react-data-grid/tree-data/ 301
 /:lang/x/react-data-grid/group-pivot/ /:lang/x/react-data-grid/tree-data/ 301
+/x/react-data-grid/columns/ /x/react-data-grid/column-definition/ 301
```

Comment:
```suggestion
/x/react-data-grid/columns/ /x/react-data-grid/column-definition/ 301
/:lang/x/react-data-grid/columns/ /:lang/x/react-data-grid/column-definition/ 301
```

Reply:
I was curious. In Portugues, we have the title translated, and the rest in English

Reply:
I was curious. In Portugues, we have the title translated, and the rest in English

Reply:
Yeah, our translations are partial. But having redirect for other languages won't hurt anyway